### PR TITLE
remove unused var, small improvements

### PR DESCRIPTION
**Pull request by Disservin**
1. remove unused imports and vars
2. use "bit count" for material, 3-4x faster than old counting method
3. take all pgns that are in the `./pgn` folder

### CODE REVIEW

**Comment by Disservin** [comment]
With 140MB of pgns the combination step took 4s.

**Comment by vondele** [comment]
1. and 2. look good to me, and I could merge them. However 3 doesn't work for me. Reading all pgns in memory only works for smaller sets of pgns. I have used the tools on more than 100GB of data. The parallelism is over files, typical collection of pgns I would use is like 100k files.

**Comment by Disservin** [comment]
Yeah okay in that case it would be a slowdown, curious to know is there really a need for that many pgns? What’s like the NormalizePawnValue diff between 1GB and 100GB?

**Comment by vondele** [comment]
To just extract the NormalizePawnValue value, I think far fewer pgns will do. I got very reasonable results with just 200k games, even few could probably be enough. A lot of games is mostly useful to get the full 2D contour plot look good.

**Comment by Disservin** [comment]
Just read 3. doesnt work for you ? `take all pgns that are in the ./pgn folder` why exactly does this not work for you?

**Comment by vondele** [comment]
This: `take all pgns that are in the ./pgn folder` will work. I was referring to loading all in memory.

**Comment by Disservin** [comment]
Ah okay. Then please have a look at the changes code ^^

File: scoreWLDstat.py [hunk 1]
```diff
@@ -101,43 +86,37 @@ def ana_pos(self, filename):
...
+    parser.add_argument(
+        "--matching_plies",
+        type=int,
+        default=6,
+        help="Number of plies that the material situation needs to be on the board (unused).",
     )
+    args = parser.parse_args()
+
+    # use all files in ./pgns that end with pgn
+    path = "./pgns"
```
Comment (vondele) [inline scoreWLDstat.py:101]: so, what about making that actually an argument of the script (the default can be ./pgns) ?

**Comment by vondele** [comment]
Thanks, works for me!

**Pull request merged**